maybe a  bug in batch_impl.go, not expect behavior with Wrapf(e, errFmt, errArgs)

<!-- Please answer these questions when reporting a bug. Thanks! -->

### What version of iotex-core image (or code branch) are you using?

### What operating system and processor architecture are you using?

### What did you do? If possible, provide a recipe for reproducing the error.

i am writing a [linter](https://github.com/alingse/asasalint) to lint that pass []any as any in variadic function

and run github Actions for some top go packages, this package failed

see github actions lint result here https://github.com/alingse/asasalint/runs/7284520322?check_suite_focus=true

```
Error: /home/runner/work/asasalint/asasalint/iotex-core/db/batch/batch_impl.go:73:51: pass []any as any to func batch func(op github.com/iotexproject/iotex-core/db/batch.WriteType, namespace string, key []byte, value []byte, errorFormat string, errorArgs ...interface{})
Error: /home/runner/work/asasalint/asasalint/iotex-core/db/batch/batch_impl.go:80:52: pass []any as any to func batch func(op github.com/iotexproject/iotex-core/db/batch.WriteType, namespace string, key []byte, value []byte, errorFormat string, errorArgs ...interface{})
Error: /home/runner/work/asasalint/asasalint/iotex-core/db/batch/batch_impl.go:263:65: pass []any as any to func batch func(op github.com/iotexproject/iotex-core/db/batch.WriteType, namespace string, key []byte, value []byte, errorFormat string, errorArgs ...interface{})
Error: /home/runner/work/asasalint/asasalint/iotex-core/db/batch/batch_impl.go:273:66: pass []any as any to func batch func(op github.com/iotexproject/iotex-core/db/batch.WriteType, namespace string, key []byte, value []byte, errorFormat string, errorArgs ...interface{})
```


### What did you expect to see?



### What did you see instead?

--- a/blockindex/indexer.go
+++ b/blockindex/indexer.go
@@ -9,6 +9,7 @@ package blockindex
 import (
 	"bytes"
 	"context"
+	"fmt"
 	"math/big"
 	"sync"
 
@@ -155,7 +156,7 @@ func (x *blockIndexer) DeleteTipBlock(ctx context.Context, blk *block.Block) err
 	}
 	// delete hash --> height
 	hash := blk.HashBlock()
-	x.batch.Delete(_blockHashToHeightNS, hash[_hashOffset:], "failed to delete block at height %d", height)
+	x.batch.Delete(_blockHashToHeightNS, hash[_hashOffset:], fmt.Sprintf("failed to delete block at height %d", height))
 	// delete from total block index
 	if err := x.tbk.Revert(1); err != nil {
 		return err
@@ -170,7 +171,7 @@ func (x *blockIndexer) DeleteTipBlock(ctx context.Context, blk *block.Block) err
 		if err != nil {
 			return err
 		}
-		x.batch.Delete(_actionToBlockHashNS, actHash[_hashOffset:], "failed to delete action hash %x", actHash)
+		x.batch.Delete(_actionToBlockHashNS, actHash[_hashOffset:], fmt.Sprintf("failed to delete action hash %x", actHash))
 		if err := x.indexAction(actHash, selp, false, fCtx.TolerateLegacyAddress); err != nil {
 			return err
 		}
@@ -336,7 +337,7 @@ func (x *blockIndexer) putBlock(ctx context.Context, blk *block.Block) error {
 		if err != nil {
 			return err
 		}
-		x.batch.Put(_actionToBlockHashNS, actHash[_hashOffset:], ad, "failed to put action hash %x", actHash)
+		x.batch.Put(_actionToBlockHashNS, actHash[_hashOffset:], ad, fmt.Sprintf("failed to put action hash %x", actHash))
 		// add to total account index
 		if err := x.tac.Add(actHash[:], true); err != nil {
 			return err

--- a/db/batch/batch.go
+++ b/db/batch/batch.go
@@ -40,9 +40,9 @@ type (
 		// ClearAndUnlock clears the write queue and unlocks the batch
 		ClearAndUnlock()
 		// Put insert or update a record identified by (namespace, key)
-		Put(string, []byte, []byte, string, ...interface{})
+		Put(string, []byte, []byte, string)
 		// Delete deletes a record by (namespace, key)
-		Delete(string, []byte, string, ...interface{})
+		Delete(string, []byte, string)
 		// Size returns the size of batch
 		Size() int
 		// Entry returns the entry at the index

--- a/db/batch/batch_impl.go
+++ b/db/batch/batch_impl.go
@@ -70,14 +70,14 @@ func (b *baseKVStoreBatch) ClearAndUnlock() {
 func (b *baseKVStoreBatch) Put(namespace string, key, value []byte, errorFormat string, errorArgs ...interface{}) {
 	b.mutex.Lock()
 	defer b.mutex.Unlock()
-	b.batch(Put, namespace, key, value, errorFormat, errorArgs)
+	b.batch(Put, namespace, key, value, errorFormat, errorArgs...)
 }
 
 // Delete deletes a record
 func (b *baseKVStoreBatch) Delete(namespace string, key []byte, errorFormat string, errorArgs ...interface{}) {
 	b.mutex.Lock()
 	defer b.mutex.Unlock()
-	b.batch(Delete, namespace, key, nil, errorFormat, errorArgs)
+	b.batch(Delete, namespace, key, nil, errorFormat, errorArgs...)
 }
 
 // Size returns the size of batch
@@ -260,7 +260,7 @@ func (cb *cachedBatch) Put(namespace string, key, value []byte, errorFormat stri
 	h := cb.hash(namespace, key)
 	cb.touchKey(h)
 	cb.currentCache().Write(&h, value)
-	cb.kvStoreBatch.batch(Put, namespace, key, value, errorFormat, errorArgs)
+	cb.kvStoreBatch.batch(Put, namespace, key, value, errorFormat, errorArgs...)
 }
 
 // Delete deletes a record
@@ -270,7 +270,7 @@ func (cb *cachedBatch) Delete(namespace string, key []byte, errorFormat string, 
 	h := cb.hash(namespace, key)
 	cb.touchKey(h)
 	cb.currentCache().Evict(&h)
-	cb.kvStoreBatch.batch(Delete, namespace, key, nil, errorFormat, errorArgs)
+	cb.kvStoreBatch.batch(Delete, namespace, key, nil, errorFormat, errorArgs...)
 }
 
 // Clear clear the cached batch buffer

--- a/db/batch/batch_impl.go
+++ b/db/batch/batch_impl.go
@@ -67,17 +67,17 @@ func (b *baseKVStoreBatch) ClearAndUnlock() {
 }
 
 // Put inserts a <key, value> record
-func (b *baseKVStoreBatch) Put(namespace string, key, value []byte, errorFormat string, errorArgs ...interface{}) {
+func (b *baseKVStoreBatch) Put(namespace string, key, value []byte, errorMessage string) {
 	b.mutex.Lock()
 	defer b.mutex.Unlock()
-	b.batch(Put, namespace, key, value, errorFormat, errorArgs...)
+	b.batch(Put, namespace, key, value, errorMessage)
 }
 
 // Delete deletes a record
-func (b *baseKVStoreBatch) Delete(namespace string, key []byte, errorFormat string, errorArgs ...interface{}) {
+func (b *baseKVStoreBatch) Delete(namespace string, key []byte, errorMessage string) {
 	b.mutex.Lock()
 	defer b.mutex.Unlock()
-	b.batch(Delete, namespace, key, nil, errorFormat, errorArgs...)
+	b.batch(Delete, namespace, key, nil, errorMessage)
 }
 
 // Size returns the size of batch
@@ -163,16 +163,15 @@ func (b *baseKVStoreBatch) AddFillPercent(ns string, percent float64) {
 }
 
 // batch puts an entry into the write queue
-func (b *baseKVStoreBatch) batch(op WriteType, namespace string, key, value []byte, errorFormat string, errorArgs ...interface{}) {
+func (b *baseKVStoreBatch) batch(op WriteType, namespace string, key, value []byte, errorMessage string) {
 	b.writeQueue = append(
 		b.writeQueue,
 		&WriteInfo{
-			writeType:   op,
-			namespace:   namespace,
-			key:         key,
-			value:       value,
-			errorFormat: errorFormat,
-			errorArgs:   errorArgs,
+			writeType:    op,
+			namespace:    namespace,
+			key:          key,
+			value:        value,
+			errorMessage: errorMessage,
 		})
 }
 
@@ -254,23 +253,23 @@ func (cb *cachedBatch) touchKey(h kvCacheKey) {
 }
 
 // Put inserts a <key, value> record
-func (cb *cachedBatch) Put(namespace string, key, value []byte, errorFormat string, errorArgs ...interface{}) {
+func (cb *cachedBatch) Put(namespace string, key, value []byte, errorMessage string) {
 	cb.lock.Lock()
 	defer cb.lock.Unlock()
 	h := cb.hash(namespace, key)
 	cb.touchKey(h)
 	cb.currentCache().Write(&h, value)
-	cb.kvStoreBatch.batch(Put, namespace, key, value, errorFormat, errorArgs...)
+	cb.kvStoreBatch.batch(Put, namespace, key, value, errorMessage)
 }
 
 // Delete deletes a record
-func (cb *cachedBatch) Delete(namespace string, key []byte, errorFormat string, errorArgs ...interface{}) {
+func (cb *cachedBatch) Delete(namespace string, key []byte, errorMessage string) {
 	cb.lock.Lock()
 	defer cb.lock.Unlock()
 	h := cb.hash(namespace, key)
 	cb.touchKey(h)
 	cb.currentCache().Evict(&h)
-	cb.kvStoreBatch.batch(Delete, namespace, key, nil, errorFormat, errorArgs...)
+	cb.kvStoreBatch.batch(Delete, namespace, key, nil, errorMessage)
 }
 
 // Clear clear the cached batch buffer

--- a/db/batch/batch_impl_test.go
+++ b/db/batch/batch_impl_test.go
@@ -64,7 +64,6 @@ func TestBaseKVStoreBatch(t *testing.T) {
 				wi.Key(),
 				wi.Value(),
 				"",
-				nil,
 			)
 		}
 		return wi

--- a/db/batch/writeinfo.go
+++ b/db/batch/writeinfo.go
@@ -19,12 +19,11 @@ type (
 
 	// WriteInfo is the struct to store Put/Delete operation info
 	WriteInfo struct {
-		writeType   WriteType
-		namespace   string
-		key         []byte
-		value       []byte
-		errorFormat string
-		errorArgs   interface{}
+		writeType    WriteType
+		namespace    string
+		key          []byte
+		value        []byte
+		errorMessage string
 	}
 
 	// WriteInfoFilter filters a write
@@ -43,16 +42,14 @@ func NewWriteInfo(
 	namespace string,
 	key,
 	value []byte,
-	errorFormat string,
-	errorArgs interface{},
+	errorMessage string,
 ) *WriteInfo {
 	return &WriteInfo{
-		writeType:   writeType,
-		namespace:   namespace,
-		key:         key,
-		value:       value,
-		errorFormat: errorFormat,
-		errorArgs:   errorArgs,
+		writeType:    writeType,
+		namespace:    namespace,
+		key:          key,
+		value:        value,
+		errorMessage: errorMessage,
 	}
 }
 
@@ -82,14 +79,9 @@ func (wi *WriteInfo) Value() []byte {
 	return value
 }
 
-// ErrorFormat returns the error format
-func (wi *WriteInfo) ErrorFormat() string {
-	return wi.errorFormat
-}
-
-// ErrorArgs returns the error args
-func (wi *WriteInfo) ErrorArgs() interface{} {
-	return wi.errorArgs
+// Error returns the error Message
+func (wi *WriteInfo) Error() string {
+	return wi.errorMessage
 }
 
 // Serialize serializes the write info

--- a/db/batch/writeinfo_test.go
+++ b/db/batch/writeinfo_test.go
@@ -8,6 +8,7 @@ package batch
 
 import (
 	"bytes"
+	"fmt"
 	"testing"
 
 	"github.com/stretchr/testify/require"
@@ -18,13 +19,12 @@ func TestWriteInfo(t *testing.T) {
 	key := []byte("key")
 	value := []byte("value")
 	ef := "fail to put %x"
-	wi := NewWriteInfo(Put, ns, key, value, ef, key)
+	wi := NewWriteInfo(Put, ns, key, value, fmt.Sprintf(ef, key))
 	require.Equal(t, ns, wi.Namespace())
 	require.Equal(t, Put, wi.WriteType())
 	require.True(t, bytes.Equal(key, wi.Key()))
 	require.True(t, bytes.Equal(value, wi.Value()))
-	require.Equal(t, ef, wi.ErrorFormat())
-	require.Equal(t, key, wi.ErrorArgs())
+	require.Equal(t, fmt.Sprintf(ef, key), wi.Error())
 	require.True(t, bytes.Equal([]byte{110, 97, 109, 101, 115, 112, 97, 99, 101, 107, 101, 121, 118, 97, 108, 117, 101}, wi.SerializeWithoutWriteType()))
 	require.True(t, bytes.Equal([]byte{0, 110, 97, 109, 101, 115, 112, 97, 99, 101, 107, 101, 121, 118, 97, 108, 117, 101}, wi.Serialize()))
 }

--- a/db/counting_index.go
+++ b/db/counting_index.go
@@ -7,6 +7,8 @@
 package db
 
 import (
+	"fmt"
+
 	"github.com/pkg/errors"
 
 	"github.com/iotexproject/iotex-core/db/batch"
@@ -119,8 +121,8 @@ func (c *countingIndex) Add(value []byte, inBatch bool) error {
 		return errors.Wrap(ErrInvalid, "cannot call Add in batch mode, call Commit() first to exit batch mode")
 	}
 	b := batch.NewBatch()
-	b.Put(c.bucket, byteutil.Uint64ToBytesBigEndian(c.size), value, "failed to add %d-th item", c.size+1)
-	b.Put(c.bucket, CountKey, byteutil.Uint64ToBytesBigEndian(c.size+1), "failed to update size = %d", c.size+1)
+	b.Put(c.bucket, byteutil.Uint64ToBytesBigEndian(c.size), value, fmt.Sprintf("failed to add %d-th item", c.size+1))
+	b.Put(c.bucket, CountKey, byteutil.Uint64ToBytesBigEndian(c.size+1), fmt.Sprintf("failed to update size = %d", c.size+1))
 	b.AddFillPercent(c.bucket, 1.0)
 	if err := c.kvStore.WriteBatch(b); err != nil {
 		return err
@@ -134,7 +136,7 @@ func (c *countingIndex) addBatch(value []byte) error {
 	if c.batch == nil {
 		c.batch = batch.NewBatch()
 	}
-	c.batch.Put(c.bucket, byteutil.Uint64ToBytesBigEndian(c.size), value, "failed to add %d-th item", c.size+1)
+	c.batch.Put(c.bucket, byteutil.Uint64ToBytesBigEndian(c.size), value, fmt.Sprintf("failed to add %d-th item", c.size+1))
 	c.size++
 	return nil
 }
@@ -166,9 +168,9 @@ func (c *countingIndex) Revert(count uint64) error {
 	b := batch.NewBatch()
 	start := c.size - count
 	for i := uint64(0); i < count; i++ {
-		b.Delete(c.bucket, byteutil.Uint64ToBytesBigEndian(start+i), "failed to delete %d-th item", start+i)
+		b.Delete(c.bucket, byteutil.Uint64ToBytesBigEndian(start+i), fmt.Sprintf("failed to delete %d-th item", start+i))
 	}
-	b.Put(c.bucket, CountKey, byteutil.Uint64ToBytesBigEndian(start), "failed to update size = %d", start)
+	b.Put(c.bucket, CountKey, byteutil.Uint64ToBytesBigEndian(start), fmt.Sprintf("failed to update size = %d", start))
 	b.AddFillPercent(c.bucket, 1.0)
 	if err := c.kvStore.WriteBatch(b); err != nil {
 		return err
@@ -189,7 +191,7 @@ func (c *countingIndex) Commit() error {
 	if c.batch == nil {
 		return nil
 	}
-	c.batch.Put(c.bucket, CountKey, byteutil.Uint64ToBytesBigEndian(c.size), "failed to update size = %d", c.size)
+	c.batch.Put(c.bucket, CountKey, byteutil.Uint64ToBytesBigEndian(c.size), fmt.Sprintf("failed to update size = %d", c.size))
 	c.batch.AddFillPercent(c.bucket, 1.0)
 	if err := c.kvStore.WriteBatch(c.batch); err != nil {
 		return err
@@ -212,7 +214,7 @@ func (c *countingIndex) Finalize() error {
 	if c.batch == nil {
 		return ErrInvalid
 	}
-	c.batch.Put(c.bucket, CountKey, byteutil.Uint64ToBytesBigEndian(c.size), "failed to update size = %d", c.size)
+	c.batch.Put(c.bucket, CountKey, byteutil.Uint64ToBytesBigEndian(c.size), fmt.Sprintf("failed to update size = %d", c.size))
 	c.batch.AddFillPercent(c.bucket, 1.0)
 	c.batch = nil
 	return nil

--- a/db/db_bolt.go
+++ b/db/db_bolt.go
@@ -302,19 +302,17 @@ func (b *BoltDB) WriteBatch(kvsb batch.KVStoreBatch) (err error) {
 					return e
 				}
 				ns := write.Namespace()
-				errFmt := write.ErrorFormat()
-				errArgs := write.ErrorArgs()
 				switch write.WriteType() {
 				case batch.Put:
 					bucket, e := tx.CreateBucketIfNotExists([]byte(ns))
 					if e != nil {
-						return errors.Wrapf(e, errFmt, errArgs)
+						return errors.Wrap(e, write.Error())
 					}
 					if p, ok := kvsb.CheckFillPercent(ns); ok {
 						bucket.FillPercent = p
 					}
 					if e := bucket.Put(write.Key(), write.Value()); e != nil {
-						return errors.Wrapf(e, errFmt, errArgs)
+						return errors.Wrap(e, write.Error())
 					}
 				case batch.Delete:
 					bucket := tx.Bucket([]byte(ns))
@@ -322,7 +320,7 @@ func (b *BoltDB) WriteBatch(kvsb batch.KVStoreBatch) (err error) {
 						continue
 					}
 					if e := bucket.Delete(write.Key()); e != nil {
-						return errors.Wrapf(e, errFmt, errArgs)
+						return errors.Wrap(e, write.Error())
 					}
 				}
 			}

--- a/db/kvstorewithbuffer.go
+++ b/db/kvstorewithbuffer.go
@@ -3,6 +3,7 @@ package db
 import (
 	"bytes"
 	"context"
+	"fmt"
 
 	"github.com/pkg/errors"
 
@@ -176,21 +177,21 @@ func (kvb *kvStoreWithBuffer) Get(ns string, key []byte) ([]byte, error) {
 }
 
 func (kvb *kvStoreWithBuffer) Put(ns string, key, value []byte) error {
-	kvb.buffer.Put(ns, key, value, "faild to put %x in %s", key, ns)
+	kvb.buffer.Put(ns, key, value, fmt.Sprintf("faild to put %x in %s", key, ns))
 	return nil
 }
 
 func (kvb *kvStoreWithBuffer) MustPut(ns string, key, value []byte) {
-	kvb.buffer.Put(ns, key, value, "faild to put %x in %s", key, ns)
+	kvb.buffer.Put(ns, key, value, fmt.Sprintf("faild to put %x in %s", key, ns))
 }
 
 func (kvb *kvStoreWithBuffer) Delete(ns string, key []byte) error {
-	kvb.buffer.Delete(ns, key, "failed to delete %x in %s", key, ns)
+	kvb.buffer.Delete(ns, key, fmt.Sprintf("failed to delete %x in %s", key, ns))
 	return nil
 }
 
 func (kvb *kvStoreWithBuffer) MustDelete(ns string, key []byte) {
-	kvb.buffer.Delete(ns, key, "failed to delete %x in %s", key, ns)
+	kvb.buffer.Delete(ns, key, fmt.Sprintf("failed to delete %x in %s", key, ns))
 }
 
 func (kvb *kvStoreWithBuffer) Filter(ns string, cond Condition, minKey, maxKey []byte) ([][]byte, [][]byte, error) {
@@ -272,9 +273,9 @@ func (kvb *kvStoreWithBuffer) WriteBatch(b batch.KVStoreBatch) (err error) {
 	for _, write := range writes {
 		switch write.WriteType() {
 		case batch.Put:
-			kvb.buffer.Put(write.Namespace(), write.Key(), write.Value(), write.ErrorFormat(), write.ErrorArgs())
+			kvb.buffer.Put(write.Namespace(), write.Key(), write.Value(), write.Error())
 		case batch.Delete:
-			kvb.buffer.Delete(write.Namespace(), write.Key(), write.ErrorFormat(), write.ErrorArgs())
+			kvb.buffer.Delete(write.Namespace(), write.Key(), write.Error())
 		default:
 			log.S().Panic("unexpected write type")
 		}

--- a/db/kvstorewithbuffer_test.go
+++ b/db/kvstorewithbuffer_test.go
@@ -99,11 +99,11 @@ func TestFlusher(t *testing.T) {
 			require.Equal(t, value, f.SerializeQueue())
 		})
 		t.Run("MustPut", func(t *testing.T) {
-			buffer.EXPECT().Put(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).Times(1)
+			buffer.EXPECT().Put(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).Times(1)
 			kvb.MustPut(ns, key, value)
 		})
 		t.Run("MustDelete", func(t *testing.T) {
-			buffer.EXPECT().Delete(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).Times(1)
+			buffer.EXPECT().Delete(gomock.Any(), gomock.Any(), gomock.Any()).Times(1)
 			kvb.MustDelete(ns, key)
 		})
 	})

--- a/test/mock/mock_batch/mock_batch.go
+++ b/test/mock/mock_batch/mock_batch.go
@@ -86,20 +86,15 @@ func (mr *MockKVStoreBatchMockRecorder) ClearAndUnlock() *gomock.Call {
 }
 
 // Delete mocks base method.
-func (m *MockKVStoreBatch) Delete(arg0 string, arg1 []byte, arg2 string, arg3 ...interface{}) {
+func (m *MockKVStoreBatch) Delete(arg0 string, arg1 []byte, arg2 string) {
 	m.ctrl.T.Helper()
-	varargs := []interface{}{arg0, arg1, arg2}
-	for _, a := range arg3 {
-		varargs = append(varargs, a)
-	}
-	m.ctrl.Call(m, "Delete", varargs...)
+	m.ctrl.Call(m, "Delete", arg0, arg1, arg2)
 }
 
 // Delete indicates an expected call of Delete.
-func (mr *MockKVStoreBatchMockRecorder) Delete(arg0, arg1, arg2 interface{}, arg3 ...interface{}) *gomock.Call {
+func (mr *MockKVStoreBatchMockRecorder) Delete(arg0, arg1, arg2 interface{}) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
-	varargs := append([]interface{}{arg0, arg1, arg2}, arg3...)
-	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "Delete", reflect.TypeOf((*MockKVStoreBatch)(nil).Delete), varargs...)
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "Delete", reflect.TypeOf((*MockKVStoreBatch)(nil).Delete), arg0, arg1, arg2)
 }
 
 // Entry mocks base method.
@@ -130,20 +125,15 @@ func (mr *MockKVStoreBatchMockRecorder) Lock() *gomock.Call {
 }
 
 // Put mocks base method.
-func (m *MockKVStoreBatch) Put(arg0 string, arg1, arg2 []byte, arg3 string, arg4 ...interface{}) {
+func (m *MockKVStoreBatch) Put(arg0 string, arg1, arg2 []byte, arg3 string) {
 	m.ctrl.T.Helper()
-	varargs := []interface{}{arg0, arg1, arg2, arg3}
-	for _, a := range arg4 {
-		varargs = append(varargs, a)
-	}
-	m.ctrl.Call(m, "Put", varargs...)
+	m.ctrl.Call(m, "Put", arg0, arg1, arg2, arg3)
 }
 
 // Put indicates an expected call of Put.
-func (mr *MockKVStoreBatchMockRecorder) Put(arg0, arg1, arg2, arg3 interface{}, arg4 ...interface{}) *gomock.Call {
+func (mr *MockKVStoreBatchMockRecorder) Put(arg0, arg1, arg2, arg3 interface{}) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
-	varargs := append([]interface{}{arg0, arg1, arg2, arg3}, arg4...)
-	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "Put", reflect.TypeOf((*MockKVStoreBatch)(nil).Put), varargs...)
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "Put", reflect.TypeOf((*MockKVStoreBatch)(nil).Put), arg0, arg1, arg2, arg3)
 }
 
 // SerializeQueue mocks base method.
@@ -275,20 +265,15 @@ func (mr *MockCachedBatchMockRecorder) ClearAndUnlock() *gomock.Call {
 }
 
 // Delete mocks base method.
-func (m *MockCachedBatch) Delete(arg0 string, arg1 []byte, arg2 string, arg3 ...interface{}) {
+func (m *MockCachedBatch) Delete(arg0 string, arg1 []byte, arg2 string) {
 	m.ctrl.T.Helper()
-	varargs := []interface{}{arg0, arg1, arg2}
-	for _, a := range arg3 {
-		varargs = append(varargs, a)
-	}
-	m.ctrl.Call(m, "Delete", varargs...)
+	m.ctrl.Call(m, "Delete", arg0, arg1, arg2)
 }
 
 // Delete indicates an expected call of Delete.
-func (mr *MockCachedBatchMockRecorder) Delete(arg0, arg1, arg2 interface{}, arg3 ...interface{}) *gomock.Call {
+func (mr *MockCachedBatchMockRecorder) Delete(arg0, arg1, arg2 interface{}) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
-	varargs := append([]interface{}{arg0, arg1, arg2}, arg3...)
-	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "Delete", reflect.TypeOf((*MockCachedBatch)(nil).Delete), varargs...)
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "Delete", reflect.TypeOf((*MockCachedBatch)(nil).Delete), arg0, arg1, arg2)
 }
 
 // Entry mocks base method.
@@ -334,20 +319,15 @@ func (mr *MockCachedBatchMockRecorder) Lock() *gomock.Call {
 }
 
 // Put mocks base method.
-func (m *MockCachedBatch) Put(arg0 string, arg1, arg2 []byte, arg3 string, arg4 ...interface{}) {
+func (m *MockCachedBatch) Put(arg0 string, arg1, arg2 []byte, arg3 string) {
 	m.ctrl.T.Helper()
-	varargs := []interface{}{arg0, arg1, arg2, arg3}
-	for _, a := range arg4 {
-		varargs = append(varargs, a)
-	}
-	m.ctrl.Call(m, "Put", varargs...)
+	m.ctrl.Call(m, "Put", arg0, arg1, arg2, arg3)
 }
 
 // Put indicates an expected call of Put.
-func (mr *MockCachedBatchMockRecorder) Put(arg0, arg1, arg2, arg3 interface{}, arg4 ...interface{}) *gomock.Call {
+func (mr *MockCachedBatchMockRecorder) Put(arg0, arg1, arg2, arg3 interface{}) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
-	varargs := append([]interface{}{arg0, arg1, arg2, arg3}, arg4...)
-	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "Put", reflect.TypeOf((*MockCachedBatch)(nil).Put), varargs...)
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "Put", reflect.TypeOf((*MockCachedBatch)(nil).Put), arg0, arg1, arg2, arg3)
 }
 
 // ResetSnapshots mocks base method.

